README.md

```diff
@@ -337,7 +337,7 @@ end up there.
 | exportCompleteIconSet     | boolean                    | false                                    | Specifies if the complete icon set should be exported or not (can be very handy for showcases)                                                                                  |
 | prefix                    | string                     | myIcon                                   | prefix for the generated svg constants                                                                                                                                          |
 | interfaceName             | string                     | MyIcon                                   | name for the generated interface                                                                                                                                                |
-| fileName                  | stirng                     | my-icons                                 | file name of the generated file                                                                                                                                                 |
+| modelFileName             | string                     | my-icons                                 | file name of the generated file                                                                                                                                                 |
```

Comment:
This one is correct. If the `conversionType` is set to `files` then we have the modelFileName property.